README.md

```diff
@@ -11,7 +11,7 @@ ___
 
 ## Try it out on Heroku
 
-[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/lexoyo/platform-client/tree/master)
```

Comment:
Hey @lexoyo could you change this to be the URL to ushahidi's repo instead of your fork's url? after that I will merged this. Thanks!